### DISCUSSION TITLE
Certificate Location

### DISCUSSION THREAD

**Discussion by mancda** [dotnet/maui#16596] [Q&A]
When configuring my iOS Bundle signing in the application properties I am finding certificates that are not in my apple account.  The list in question appears when looking at the manual provisioning certificate options.  Where are these other certificates being stored by VS? And is it possible to edit this list so I can remove one that is expired.  I have removed and readded my apple account under options but to no avail.

Thanks for your help

**Comment by faheys** [comment]
On a Windows machine mine are located at C:\Users\\[username]\AppData\Local\Xamarin\iOS\Provisioning\Certificates
